test-integration/operator-tests/src/test/java/org/infinispan/operator/CustomLibsIT.java

```diff
@@ -0,0 +1,107 @@
+package org.infinispan.operator;
+
+import cz.xtf.core.config.XTFConfig;
+import cz.xtf.core.openshift.OpenShift;
+import cz.xtf.core.openshift.OpenShiftWaiters;
+import cz.xtf.core.openshift.OpenShifts;
+import cz.xtf.junit5.annotations.CleanBeforeAll;
+import io.fabric8.kubernetes.api.model.*;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.maven.it.VerificationException;
+import org.apache.maven.it.Verifier;
+import org.assertj.core.api.Assertions;
+import org.infinispan.Infinispan;
+import org.infinispan.Infinispans;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.Collections;
+
+@Slf4j
+@CleanBeforeAll
+public class CustomLibsIT {
+    private static final OpenShift openShift = OpenShifts.master();
+    private static final OpenShiftWaiters openShiftWaiters = OpenShiftWaiters.get(openShift, () -> false);
+    private static final Infinispan infinispan = Infinispans.customLibs();
+
+    @BeforeAll
+    static void prepare() throws IOException {
+        preparePVC();
+        preparePod();
+
+        openShiftWaiters.areExactlyNPodsRunning(1).waitFor();
+
+        buildLibs();
+        uploadLibs();
+
+        infinispan.deploy();
+        infinispan.waitFor();
+    }
+
+    @Test
+    void test() {
+        Pod pod = openShift.getPod("custom-libs-0");
+        String logs = openShift.getPodLog(pod);
+
+        Assertions.assertThat(logs).contains("Loaded extension 'basic-filter-factoy'");
+    }
+
+    private static void preparePVC() {
+        PersistentVolumeClaimBuilder pvc = new PersistentVolumeClaimBuilder();
+        pvc.withNewMetadata().withName("infinispan-libs").endMetadata();
+        pvc.withNewSpec()
+                .withAccessModes("ReadWriteOnce")
+                .withNewResources().withRequests(Collections.singletonMap("storage", new Quantity("100Mi")))
+                .endResources().endSpec();
+
+        openShift.createPersistentVolumeClaim(pvc.build());
+    }
+
+    private static void preparePod() {
+        String serverImage = XTFConfig.get("org.infinispan.image.server", "quay.io/infinispan/server:latest");
```

Comment:
@Crumby do you really need to boot Pod with Infinispan sever for file exchange? I think `busybox` will be enough.

Reply:
Probably not, I just folled up what is documented. I was trying `ubi8` image but I had issues with that one and even if I suceded then ubi8 does not contain tar so `oc cp ...` would work anyway. `busybox` may be better.

Reply:
@dmvolod I tried `busybox` but it suffers from the same issue as `ubi8`: endless restart. It could be fixed by adding some manual sleep cmd to the pod but we need to consider that this tests are going to be executed on different architectures. And busybox is not available for s390x or ppc.

Reply:
Oh, yeah, @Crumby good point, let's keep as is. Probably we can play with entry-point to avoid whole Infinispan server boot on startup

Reply:
One more question, @Crumby do you know, why `busybox` is not available on `s390x`. Docker hub says it's present for that archs.

Reply:
@dmvolod You are correct, it seems I've been looking at wrong page.